package.json

```diff
@@ -13,7 +13,7 @@
     "publish-docs": "gh-pages -d docs/jsdocs",
     "start:test": "gulp dev:test",
     "build:test": "gulp build:test",
-    "test": "yarn audit && yarn install-test-deps && yarn test:unit && yarn lint",
+    "test": "node scripts/audit.js && yarn install-test-deps && yarn test:unit && yarn lint",
```

Comment:
could we maybe have the local test command run `yarn audit` as usual and only have travis run the version with the 10 threshold?

Reply:
we could also put it last in the test command so that it's not blocking the other tasks from running

Reply:
@diracdeltas done 👍